packages/messaging/src/interfaces/message-payload.ts

```diff
@@ -29,5 +27,5 @@ export interface FcmOptions {
 export interface MessagePayload {
   fcmOptions?: FcmOptions;
   notification?: NotificationDetails;
-  data?: { [key: string]: unknown };
+  data?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
Isn't object better than any, in that it forces you to check whether anything is there rather than silently becoming anything?

Reply:
Good point. Made it `unknown`.